stl/inc/type_traits

```diff
@@ -1473,6 +1473,14 @@ _NODISCARD constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_c
     return _STD move(_Arg);
 }
 
+// Currently, invoke() isn’t depicted as constexpr in the Standard, and implementers are forbidden from “strengthening”
+// it to be constexpr, yet variant visit() is required to be constexpr and have invoke-like behavior. We solve this by
+// using a macro to stamp out a public non-constexpr and internal constexpr implementation.
+
+// Now that a Core Language issue affecting constexpr invoke() has been fixed (CWG1581), we may make this unconditional
+// and remove the macro (which would technically violate the strengthening rule in C++14/17 mode).
+// TRANSITION, P1065R2

```

Comment:
This TRANSITION should be next to the bit that would have to be changed; I don't think it has anything to do with reference_wrapper or CONCAT?

Reply:
agree on reference wrapper, but we'd probably remove concat and concatx (actually should we undef them after we undef implement_invoke? They don't appear to be used later in the file

Reply:
`<xatomic.h>` uses `_CONCAT`.

Reply:
I'm OK with having a `TRANSITION` not directly attached to affected code, since the comment explains what will need to change.